docs/content/components/overlay/dialog/dialog-from-menu.demo.tsx

```diff
@@ -32,10 +32,10 @@ export default () => {
                 be undone.
               </Dialog.Content>
               <Dialog.Actions>
-                <Button size="small" variant="secondary" onPress={close}>
+                <Button variant="secondary" slot="close">
                   Cancel
                 </Button>
-                <Button size="small" variant="primary" onPress={close}>
+                <Button variant="primary" onPress={close}>
```

Comment:
The `onPress={close}` isn't here necessary or? So maybe as example and alert like `onPress={() => alert('deleted!'}`

Reply:
but the modal will not close then 🤔

Reply:
ok true. In Storybook and also in the documentation is no example of doing an action with the primary button and then close the dialog 🤔 At least we should add one or?

Reply:
I thought this is clear 😄